### DISCUSSION TITLE
今天我买了一个烤面包机，然后事情开始变得不对劲了

### DISCUSSION THREAD

**Discussion by Tragedysaver1024** [share121/inter-knot#4870] [其他]
事情是这样的：
今天我打算去旧货市场买一个烤面包机送我女友来着，然后出事了。我跟我女友兴高采烈的说起来这件事的时候聊到一半就说我我聊天奇奇怪怪，提到烤面包机的时候一直用“我”，真是奇怪，我明明是在用正常的方式来描述我这台烤面包机……
鬼知道我是有什么奇怪的地方，明天得把我拆开来看看

**Comment by Dave-12138** [comment]
看来是模因身了<img src="https://bbs-static.miyoushe.com/static/2023/12/19/ecfc14f48484637539d7eecfb48b1178_5785018828453527028.png" alt="雪衣 无言" width="75" >
